jenkins-job-generator/definitions/sapmachine-build-jobs.yml

```diff
@@ -236,10 +236,23 @@
                                 }}
                             }}
                         }}
+            sign_and_notarize: &sign_and_notarize |-
+                stage("Sign and notarize artefacts") {{
+                            when {{
```

Comment:
No when condition needed since the step should only materialized in release builds

Reply:
done